core/new-gui/src/app/workspace/component/navigation/navigation.component.ts

```diff
@@ -429,6 +431,9 @@ export class NavigationComponent {
       });
   }
 
+  onClickGetAllVersions() {
+    this.workflowVersionService.highlightVersionsDisplay();
```

Comment:
I would suggest we don't name this as "highlight", since "highlight" in the workspace context usually means highlighting an operator on the jointjs paper. and this highlight becomes a state of the workspace or a state of the highlighted link or operator. 

Here I assume is just a button is clicked, so more like an on/off toggle senario. Similar to `operator cache` button, where it can be clicked to be one state, and clicked again for another state. I hope to distinguish this scenario from "highlight" if possible.